### PR TITLE
add tip for alt attribute for the add-images-to-your-website challenge

### LINKED ISSUE #16050: Alt attribute in img not recognized
#### Challenge Name

[Add Images to your Website](https://www.freecodecamp.org/challenges/add-images-to-your-website)

#### Issue Description

Tests do not recognize the `alt` attribute in the `img` element (see code below). I think it has something to do with the hyphen? Seems to work fine with an `alt` that doesn't have it.

#### Browser Information

* Browser Name, Version: Chrome 61.0.3163.100
* Operating System: Windows 10
* Mobile, Desktop, or Tablet: Desktop

#### Your Code

```js

<link href="https://fonts.googleapis.com/css?family=Lobster" rel="stylesheet" type="text/css">
<style>
  .red-text {
    color: red;
  }

  h2 {
    font-family: Lobster, Monospace;
  }

  p {
    font-size: 16px;
    font-family: Monospace;
  }
</style>

<h2 class="red-text">CatPhotoApp</h2>

<img src="https://bit.ly/fcc-relaxing-cat" alt="Cute upside-down kitten. ">

<p class="red-text">Kitty ipsum dolor sit amet, shed everywhere shed everywhere stretching attack your ankles chase the red dot, hairball run catnip eat the grass sniff.</p>
<p class="red-text">Purr jump eat the grass rip the couch scratched sunbathe, shed everywhere rip the couch sleep in the sink fluffy fur catnip scratched.</p>

```
#### Screenshot

fcc

### PR DESCRIPTION
**Pull request by angelfeliz**
#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of freeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.org/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] All new and existing tests pass the command `npm test`. Use `git commit --amend` to amend any fixes.

#### Type of Change

- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

#### Checklist:

- [x] Tested changes locally.
- [x] Addressed currently open issue (replace XXXXX with an issue no in next line)

Closes #16050

#### Description
Add description tip for use of the alt attribute use inside the challenge

### CODE REVIEW

File: seed/challenges/01-responsive-web-design/basic-html-and-html5.json [hunk 1]
```diff
@@ -619,6 +619,7 @@
         "<code>&#60img src=\"https://www.your-image-source.com/your-image.jpg\"&#62</code>",
         "Note that in most cases, <code>img</code> elements are self-closing.",
         "All <code>img</code> elements <strong>must</strong> have an <code>alt</code> attribute. The text inside an <code>alt</code> attribute is used for screen readers to improve accessibility and is displayed if the image fails to load.",
+        "Ideally the alt attribute should not contain special chars unless needed.",
```
Comment (raisedadead) [inline seed/challenges/01-responsive-web-design/basic-html-and-html5.json]: Please wrap the `alt` around `code` tags

```diff
- "Ideally the alt attribute should not contain special chars unless needed.",
+ "Ideally the <code>alt</code>attribute should not contain special chars unless needed.",
```

**Review by raisedadead — CHANGES_REQUESTED** [review]
LGTM, except for a minor change.

**Comment by angelfeliz** [comment]
add your suggestion to the request

**Review by raisedadead — APPROVED** [review]

**Comment by raisedadead** [comment]
#### :sparkles: :sparkles: **LGTM.** :sparkles: :sparkles: 

Thanks @angelfeliz , for this pull request and contributing to this code-base for the first time.

Happy contributing!

**Pull request merged**